specification/schemaregistry/data-plane/readme.md

```diff
@@ -53,5 +53,5 @@ These settings apply only when `--tag=package-2023-07-01` is specified on the co
 
 ```yaml $(tag) == 'package-2023-07-01'
 input-file:
-  - Microsoft.EventHub/stable/2023-07-01/schemaregistry.json
+  - Microsoft.SchemaRegistry/stable/2023-07-01/schemaregistry.json
```

Comment:
@johanste - Since we're "overwriting" the existing latest swagger with this one, is this the correct procedure?
1) Remove only the latest version swagger (2023-07-01) from `Microsoft.EventHub`.
2) Emit the 2023-07-01 swagger from tsp to the `Microsoft.SchemaRegistry` folder.
3) Update only the 2023-07-01 version in the readme to point to `Microsoft.SchemaRegistry`.

Reply:
OOB discussion: remove EH folder, re-emit all swagger under Microsoft.SchemaRegistry